lib/verbalize/version.rb

```diff
@@ -1,3 +1,3 @@
 module Verbalize
-  VERSION = '2.3.0'.freeze
+  VERSION = '2.3.1'.freeze
```

Comment:
As this change is fully backwards-compatible and introduces no change to the gem's API, I went with a patch increment.